charts/rancher-thanos/v0.0.1/values.yaml

```diff
@@ -0,0 +1,156 @@
+# Default values for thanos.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+nameOverride: ""
+fullnameOverride: ""
+
+# column split Ids, e.g., c-abc:c-def
+clusterIds: ""
+
+rancherHostname: ""
+
+privateCa: false
+
+token: ""
+apiToken: ""
+
+
+grafana:
+  enabled: false
+  level: cluster
```

Comment:
Please clean up answers if they are not used in chart.
I think `level` is not used